### ISSUE TITLE
[NPC] Mikhail gives out part 13 of "The Missing Diplomat" Quest chain before part 12 is completed

### ISSUE THREAD

**Issue by wrathofthebug** [dalaranwow/dalaran-wow#5520]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
For any quest chain, the next quest is only available after the preceding quest has been completed. However, in "The Missing Diplomat" quest chain, after Slim is subdued in part 11, part 13 immediately becomes available to players from Mikhail. That should not be the case. The problem is further compounded by inaccurate scripting of Slim's behaviour in part 11 where he is subdued by players as he just turns friendly then runs off and disappears for a good 15 minutes or more before respawning in the inn with part 12 of the quest available.

**Expected behaviour**: Tell us what should happen instead
Players should need to talk to Slim first to initiate part 12 which ends at Mikhail who then gives out part 13 of the quest chain.

**Steps to reproduce the problem**:

1. Complete part 11 of the quest chain where Slim is subdued and notice that he just runs away and disappears
2. Go back to the inn and complete part 11 with Mikhail.
3. Notice that immediately part 13 is available from Mikhail while Slim is still missing/yet to respawn in the inn.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
http://wowwiki.wikia.com/wiki/Quest:The_Missing_Diplomat_(11)
http://wowwiki.wikia.com/wiki/Quest:The_Missing_Diplomat_(12)
http://wowwiki.wikia.com/wiki/Quest:The_Missing_Diplomat_(13)

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
